src/Registry.ts

```diff
@@ -32,11 +32,11 @@ export default class Registry<T> {
 	 * @returns the matching value, or a default value if one exists.
 	 */
 	match(...args: any[]): T {
-		let entries = this._entries.slice(0);
+		const entries = this._entries ? this._entries.slice(0) : [];
```

Comment:
Not an issue with this change but seeing this we should be using `import { from as arrayFrom } from 'dojo-shim/array'` here...